src/createServer.js

```diff
@@ -1,3 +1,45 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+'use strict';
+
+const http = require('http');
+const { getDataURL } = require('./convertToCase/getDataURL');
+const { getErrors } = require('./convertToCase/getErrors');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+    res.statusCode = 200;
+    res.statusMessage = 'OK';
+
+    const [originalText, targetCase] = getDataURL(req.url);
+
+    const errors = getErrors(originalText, targetCase);
+
+    if (errors.length) {
+      res.statusCode = 400;
+      res.statusMessage = 'Bad request';
+
+      res.end(JSON.stringify({ errors }));
+
+      return;
```

Comment:
looks like redundant return, after res.end method

Reply:
nope, it doesn't work well without `return`, because `res.end` after errors block work as well